neural_compressor/adaptor/tensorflow.py

```diff
@@ -83,7 +83,7 @@ def __init__(self, framework_specific_info):
 
         from pkg_resources import parse_version
         import tensorflow as tf
-        self.new_api = parse_version(tf.version.VERSION) == parse_version('2.11.0202242')
+        self.new_api = parse_version(tf.version.VERSION) == parse_version('2.11.0202250')
         self.qdq_enabled = self.itex_mode or self.format == 'QDQ' or self.new_api
```

Comment:
are those old release binaries able to get from external? if yes, shall we maintain several versions for that?

Reply:
I think they are still internal release binaries.

Reply:
the first one should have been public, right? just the latter is internal state. looks like we can hold on this PR at first.

Reply:
Can we have a version range, otherwise we couldn't test with latest spr-base feature in validation.

Reply:
I have added a spr-base supported version tuple. This PR can't hold on since the pre-CI was affected by the latest spr-base binary release.